src/main/java/graphql/annotations/EnhancedExecutionStrategy.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2016 Yurii Rashkovskii
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ */
+package graphql.annotations;
+
+import graphql.ExecutionResult;
+import graphql.execution.*;
+import graphql.language.*;
+import graphql.schema.GraphQLEnumType;
+import graphql.schema.GraphQLFieldDefinition;
+import graphql.schema.GraphQLObjectType;
+
+import java.util.HashMap;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+
+public class EnhancedExecutionStrategy extends AsyncSerialExecutionStrategy {
+
+    private static final String CLIENT_MUTATION_ID = "clientMutationId";
```

Comment:
why do you have this const?

Reply:
Sorry I don't understand the question ? The const was not introduced in this PR but in https://github.com/graphql-java/graphql-java-annotations/commit/f39d26f , and is still used in 3 places in the class.